### PR TITLE
Adding the Code of Conduct

### PR DESCRIPTION
**Pull request by AgostinoPurificato**
Inserted the code of conducted as written into dati-semantic-backend repo.

## Description

This PR does: adding the code of conduct into the repository, extracted from the "dati-semantic-backend" one.

@Clou-dia @mauropratesi

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
-# Insert code of conduct here
-# When in doubt, checkout https://www.contributor-covenant.org/
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Developers Italia is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```
Comment (bfabio) [inline CODE_OF_CONDUCT.md:5]: We might want to drop the mentions to "Developers Italia" and use something else. "this project"?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,2 +1,84 @@
...
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in Developers Italia to help us create safe and positive experiences for everyone.
```
Comment (bfabio) [inline CODE_OF_CONDUCT.md:9]: Here as well.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,2 +1,84 @@
...
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## 7. Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. You can do so by emailing contatti@developers.italia.it.
```
Comment (bfabio) [inline CODE_OF_CONDUCT.md:60]: Here

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -1,2 +1,84 @@
...
+
+## 9. Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
+
+This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+
+## 10. Contact info
+
+You can reach the managers of Developers Italia at contatti@developers.italia.it.
```
Comment (bfabio) [inline CODE_OF_CONDUCT.md:76]: And here.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -1,2 +1,84 @@
...
+* Inappropriate photography or recording.
+* Inappropriate physical contact. You should have someone's consent before touching them.
+* Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
+* Deliberate intimidation, stalking or following (online or in person).
+* Advocating for, or encouraging, any of the above behavior.
+* Sustained disruption of community events, including talks and presentations.
+
+## 5. Weapons Policy
+
+No weapons will be allowed at Developers Italia events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon. Community members are further expected to comply with all state and local laws on this matter.
```
Comment (bfabio) [inline CODE_OF_CONDUCT.md:48]: Here

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (bfabio) [reply CODE_OF_CONDUCT.md:60]: @Clou-dia should we use an ISTAT owned email address here and below?

Comment (AgostinoPurificato) [reply CODE_OF_CONDUCT.md:60]: @Clou-dia @bfabio can we use _servicedesk-schema@istat.it_?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Clou-dia) [reply CODE_OF_CONDUCT.md:5]: yes, fine for me to say "this project"

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (Clou-dia) [reply CODE_OF_CONDUCT.md:60]: yes

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (AgostinoPurificato) [reply CODE_OF_CONDUCT.md]: Resolved with new commit.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (AgostinoPurificato) [reply CODE_OF_CONDUCT.md]: Resolved with new commit.

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (AgostinoPurificato) [reply CODE_OF_CONDUCT.md]: Resolved with new commit.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (AgostinoPurificato) [reply CODE_OF_CONDUCT.md]: Resolved with new commit.

**Review by bfabio — APPROVED** [review]
I guess these changes should bounce back to the `dati-semantic-backend' repository as well.

**Pull request merged**